contracts/pool/src/contract.rs

```diff
@@ -189,11 +189,6 @@ impl LiquidityPoolTrait for LiquidityPool {
             panic_with_error!(&env, ContractError::TokenABiggerThanTokenB);
         }
 
-        if !(0..=10_000).contains(&swap_fee_bps) {
-            log!(&env, "Pool: Initialize: Fees must be between 0 and 100%");
-            panic_with_error!(&env, ContractError::InvalidBps);
-        }
-
```

Comment:
> we don't need the removed 2 verification of swap_fee_bps as this is already checked with validate_bps! at the start of the initialization.

Meaning this one should actually stay?

Reply:
no, that's okay to be gone, the first macro `validate_bps!` checks the same thing, this is redundant